### DISCUSSION TITLE
how to stop the sidebar from always opening by default

### DISCUSSION THREAD

**Discussion by thiswillbeyourgithub** [open-webui/open-webui#4426] [Q&A]
Hi,

I use vertical screens (especially relevant for reading chats!) and when I open the app the sidebar is always opened by default. This is annoying because creating a new chat opens it and making the app fullscreen opens it.

Is there a way to stop the sidebar from automatically opening?

Edit: It notably triggers when I fullscreen the app (I'm using vertical screen) but also when I put my tablet in landscape mode (I want to see more not less!)

**Comment by GrayXu** [comment]
CTRL + SHIFT + S

**Reply by thiswillbeyourgithub** [reply]
That's for closing it once. My issue is that the sidebar re opens too often
